login-workflow/src/new-architecture/screens/SuccessScreen/SuccessScreenBase.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Divider } from '@mui/material';
+import { WorkflowCard, WorkflowCardActions, WorkflowCardBody, WorkflowCardHeader } from '../../components/WorkflowCard';
+import { FinishState } from '../../../components';
```

Comment:
If you're going to use this `FinishState` component, be sure to copy it and pull it into `new-architecture`. The goal is to be able to just pluck out the `new-architecture` folder at the end with all of the things we'll need for the workflow. The exception to that rule would be things that may end up in auth-shared.